draft-ietf-tls-tls13.md

```diff
@@ -614,6 +614,11 @@ draft-21
 - Add a per-ticket nonce so that each ticket is associated with a
   different PSK (*).
 
+- Clarify that clients should change to sending with handshake keys
+  immediately upon reading ServerHello. Arguably a wire format change. (*).
```

Comment:
It either is, or it isn't.  I think that it isn't, and this is merely a clarification.

Also, this formulation doesn't work for 0-RTT at all.  The fact is that the client's second flight uses handshake keys, except for EOED.